src/psyclone/psyir/frontend/fparser2.py

```diff
@@ -4895,7 +4895,14 @@ def _parenthesis_handler(self, node, parent):
         # Use the items[1] content of the node as it contains the required
         # information (items[0] and items[2] just contain the left and right
         # brackets as strings so can be disregarded.
-        return self._create_child(node.items[1], parent)
+        new_node = self._create_child(node.items[1], parent)
+
+        # Explicit parenthesis on BinaryOperations are sometimes needed for
```

Comment:
Wow, that was very easy :-)

Reply:
It's probably worth putting a bit more detail in the comment. e.g. "...reproducibility (because a Fortran compiler may evaluate any mathematically-equivalent expression, provided that the integrity of parentheses is not violated)". This is from paragraph 2 of section 7.1.5.2.4 "Evaluation of numeric intrinsic operations" of the Fortran2008 standard.

Reply:
Done